hvac_services/data/product_product.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_product_installation" model="product.product">
+        <field name="product_tmpl_id" ref="product_template_installation"/>
+    </record>
+    <record id="product_product_maintenance" model="product.product">
+        <field name="product_tmpl_id" ref="product_template_maintenance"/>
+    </record>
+    <record id="product_product_repair" model="product.product">
+        <field name="product_tmpl_id" ref="product_template_repair"/>
+    </record>
+    <record id="appointment_account_payment.default_booking_product" model="product.product" forcecreate="1">
+        <field name="product_tmpl_id" ref="appointment_account_payment.default_booking_product_product_template"/>
+    </record>
+    <record id="product_product_4" model="product.product">
+        <field name="product_tmpl_id" ref="product_template_4"/>
+        <field name="standard_price">100.0</field>
+    </record>
+    <record id="industry_fsm_sale.field_service_product" model="product.product" forcecreate="1">
+        <field name="product_tmpl_id" ref="industry_fsm_sale.field_service_product_product_template"/>
+    </record>
```

Comment:
This is useless. Either the product template exists, and the product product as well. Either it doesn't and this will fail because you miss required fields to create a new product. I'd just drop that part
```suggestion
```